include/qengine_opencl.hpp

```diff
@@ -75,13 +79,19 @@ class QEngineOCL : public QEngine {
      * object to zero norm.
      *
      * "devID" is the index of an OpenCL device in the OCLEngine singleton, to select the device to run this engine on.
-     * "partialInit" is usually only set to true when this object is one of several collected in a
-     * Qrack::QEngineOCLMulti object, in which case this Qrack::QEngineOCL object might not contain the amplitude of the
-     * overall permutation state of the combined object.
+     * If "useHostMem" is set false, (as by default,) the QEngineOCL will attempt to allocate the state vector object
```

Comment:
Extra comma

Reply:
Let's split the difference and remove the parentheses.